wafer/kv/tests/test_kv_api.py

```diff
@@ -0,0 +1,213 @@
+"""Tests for wafer.kv api views."""
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.test import Client, TestCase
+from rest_framework.test import APIClient
+from wafer.kv.models import KeyValue
+import json
+
+
+def get_group(group):
+    return Group.objects.get(name=group)
+
+
+def create_group(group):
+    grp = Group(name=group)
+    grp.save()
+    return grp
+
+
+def create_user(username, groups):
+    create = get_user_model().objects.create_user
+    user = create(
+        username, '%s@example.com' % username, 'password')
+    for the_group in groups:
+        grp = get_group(the_group)
+        user.groups.add(grp)
+    user.save()
+    return user
+
+
+def create_kv_pair(name, value, group):
+    group = get_group(group)
+    kv = KeyValue(key=name, value=value, group=group)
```

Comment:
Similar comment about perhaps writing this as `return KeyValue.objects.create(...)`.